src/main/scala/leon/datagen/VanuatooDataGen.scala

```diff
@@ -41,16 +41,23 @@ class VanuatooDataGen(ctx: LeonContext, p: Program) extends DataGenerator {
     (n, d) -> Constructor[Expr, TypeTree](List(), RealType, s => FractionalLiteral(n, d), "" + n + "/" + d)
   }).toMap
 
+  val strings = (for (b <- Set("", "a", "\"\t\n", "Abcd")) yield {
```

Comment:
Yes it is specific to the kind of strings I want to deal with.
But there is almost no difference between `"foo"` and `"bar"`. I reverted to them in the next commit, but consider this:  
Numbers 0, 1, 2 and even 3 are very different, because they could encode size of trees during enumeration.
Similarly, using very different strings helps to identify differences between programs, whereas most programs will deal with `"foo"` and `"bar"` the same way - unless there is an indexOf("o") somewhere.
So here is why my choice:
1) Empty string
2) A single char string
3) A string with at least one capitalized letter
4) A multi-char string with non-alphanumeric chars, especially those which might need escaping.